spec/unit/bootstrap/definitions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FHIR::Definitions do
+  context '.load_extensions'
```

Comment:
This is here to note that, even though SimpleCov lists it as covered by RSpec, it isn't actually tested here. :)